README.md

```diff
@@ -10,6 +10,8 @@
 ## Getting Started
 
 ```rust
+use std::sync::Arc;
+
 use async_executors::TokioTpBuilder;
 use opentelemetry::trace::Tracer;
 use opentelemetry::global::shutdown_tracer_provider;
```

Comment:
The other thing that screwed me over was that `cargo add opentelemetry` fetched 0.17 rather than 0.16, so I spent a few minutes staring at errors relating to the the Tracer trait, before I worked out what was happening. Would it be reasonable to `pub use opentelemetry` and make the example import the re-exported crate?

```suggestion
use opentelemetry_honeycomb::opentelemetry::trace::Tracer;
use opentelemetry_honeycomb::opentelemetry::global::shutdown_tracer_provider;
```

Reply:
this sounds reasonable!